src/controller/check-your-requests.controller.ts

```diff
@@ -40,8 +41,10 @@ export const post = async (req: Request, res: Response, next: NextFunction) => {
 
         await client.gitHub.postIssue(url, body);
 
-        // TODO: Implement logic to gather the users email
-        await confirmationEmail(config.NOTIFY_USER_EMAIL, id);
+        const userEmail = getUserEmail(req);
+        if (userEmail) {
```

Comment:
This check is always true based on the control in the getUserEmail(...) method. I would suggest handling the logic in the confirmationEmail(...) method.
I would have just remove the comment here.